_infra/helm/locust/locustfiles/ce_event_details.csv

```diff
@@ -1,2 +1,2 @@
 survey_code,survey_period,mps,go_live,ref_period_start,ref_period_end,return_by,reminder,reminder2,reminder3,employment,exercise_end
-139,1806,130620,180620,150620,300620,270620,030722,230722,060822,150622,310722
+139,1806,130620,180620,150620,300620,270620,030722,230722,060822,150622,310732
```

Comment:
Extended exercise end for another 10 years as it hasn't been visible in the tests since July 2022 :o)